rezerwacje-app/src/main/resources/application.properties

```diff
@@ -8,7 +8,7 @@ spring.datasource.password=${POSTGRES_PASSWORD}
 
 # JPA
 spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
Do we want to update it for permanently and push to develop? Or it was just for necessary for this issue?

Reply:
Right, that was just for the tests. I'll drop it.